src/components/A11yStatus/a11y-status.module.scss

```diff
@@ -0,0 +1,13 @@
+.table {
+  margin-top: $spacing-06;
+}
+
+:global(.page-table .cds--tag) {
+  margin-left: 0;
+  white-space: nowrap;
+
+  span {
+    //override to fix alignment issue with variable font
```

Comment:
TODO: check if we have latest version. alignment issues should have been fixed https://github.com/IBM/plex/issues/377

Reply:
I can't remember how we're bringing in Plex, but the latest version binary is too large to update. We might need to pull it in via Akamai instead for the time being.

Reply:
Looks like they are being pulled directly from the theme.
https://github.com/carbon-design-system/gatsby-theme-carbon/tree/main/fonts
https://github.com/carbon-design-system/gatsby-theme-carbon/blob/main/packages/gatsby-theme-carbon/src/styles/internal/_plex.scss

We could probably update that to pull from Akamai instead? I'll take a look next week.